Documentation/General/productVision.tex

```diff
@@ -0,0 +1,155 @@
+MoodCat has several selling points that would make it unique in the current music market.
+
+\begin{enumerate}
+\item MoodCat is self-maintaining, which makes it possible to handle audience sizes from one to one thousand.
+This makes it possible to use the system no matter how many other users are currently online.
+
+\item MoodCat offers users both to experience solely or communicate with fellow music listeners.
+
+\item The system already works without this user input as specific interpretation of audio features allow us to find similar songs.
+The system is able to offer relevant songs using the specified current mood of the user.
+
+\item With MoodCat unknown/lesser known artists have equal chances to let their songs be played.
+This is a result of the system choosing songs based on features and is therefore unbiased regarding play counts.
+
+\item MoodCat can determine mood trends of a long-term user and can adjust the room-ordering mechanism accordingly.
+Therefore long-term users are served more suitable rooms which increases their overall satisfaction.
+
+\item MoodCat can suggest users who are repeatedly not satisfied with the songs played in a room, a different and more suitable room.
+\end{enumerate}
+
+\section{Which components should MoodCat have?}
+MoodCat will be developed as three seperate componentents: the frontend, the backend and a special music matching algorithm.
+
+\begin{enumerate}
+\item The frontend will be the visible part for the listener.
+After logging in to our services, the user will be able to select his mood and based on that MoodCat will provide several music listening rooms.
+In these rooms the user can interact with other listeners through the chat while listening to the music.
+
+\item The backend keeps track of which rooms are active and what song is currently played in the room.
+Furthermore, the backend is responsible for transmitting the chat messages between the users in the room.
+
+\item The music matching algorithm is used to compare and match moods of songs and chatrooms.
+It has two roles: list the room suggestions for a user and provide the next song to be played for a room.
+\end{enumerate}
+
+\newpage
+
+\subsection{Frontend}
+
+The MoodCat frontend will be web-based. This allows us to support all modern devices which enables a large target audience. We will develop the frontend using modern techniques as HTML5, CSS3 and AngularJS. Users can login using their Soundcloud account. This allows us to integrate with the Soundcloud platform and prevents having to register for our services first.
+
+\subsection{Backend}
+
+The MoodCat backend will be a RESTful API written in Java using JAX-RS. It will be backed by a relational (PostgreSQL) database. The backend connects the frontend to the music matching algorithm.
+
+\subsection{Music matching algorithm}
+
+During our research about mapping music to emotions/moods, we discovered the valence/arousal graph (see figure \ref{fig:avgraph}).
```

Comment:
We discovered that moods and emotions can be mapped to a valence/arousal space. Insert citation.